README.md

```diff
@@ -127,6 +173,43 @@ On the Camera Preview Page, the same second button will toggle between
 the full resolution camera view and a one-to-one pixel view intended
 to aid in focusing the camera (the camera is focused manually).
 
+# Dencam Pages
+
+The Dencam user interface has 5 pages, which are traversed using the 
+top button. Each page is displaying information designated to itself.
+(i.e. Solar Display Page displays information recieved from Sun Saver)
+
+## Networking Information Page
+
+This page displays Raspberry Pi username, wLan0 IP address, and the name of 
+the wifi signal it is connected to.
+
+## Recording Status Page
+
+Displays the current number of video recordings saved and
+stores, the file directory they are currently being stored to, the 
+amount of available storage, time stamp, and a countdown for the 
+recording to start.
```

Comment:
perhaps it should be explained that the count down only pertains after the DenCam code is first launched.  And that the second button will toggle on and off recording and that whether the device is actively recording is displayed in the same area that the countdown was.